values-schema.yaml

```diff
@@ -1280,23 +1286,25 @@ properties:
               - production
               - staging
             default: production
-        oneOf:
-          - properties:
-              issuer:
-                const: letsencrypt
-            required:
-              - email
-              - issuer
-          - allOf:
-              - oneOf:
-                  - not:
-                      required:
-                        - issuer
-                  - properties:
-                      issuer:
-                        const: custom-ca
-                    required:
-                      - issuer
+        required: [email]
```

Comment:
Is email required for customCA?

Reply:
no, fixed as this was tryout stuff that didn't get finalized